File: src/targets/javascript/axios/fixtures/nested.js
import axios from 'axios';

const options = {
  method: 'GET',
  url: 'https://httpbin.org/anything',
  params: {'foo[bar]': 'baz,zap', fiz: 'buz', key: 'value'}
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}

File: src/targets/ruby/faraday/fixtures/nested.rb
require 'faraday'

conn = Faraday.new(
  url: 'https://httpbin.org',
)

response = conn.get('/anything') do |req|
  req.params['foo[bar]'] = 'baz,zap'
  req.params['fiz'] = 'buz'
  req.params['key'] = 'value'
end

puts response.status
puts response.body